regression/cbmc/Pointer_Arithmetic18/main.c

```diff
@@ -0,0 +1,11 @@
+#define MB 0x00100000
+#define BASE (15 * MB)
+#define OFFSET (1 * MB)
+
+main()
+{
+  char *base = BASE;
+  int offset = OFFSET;
+  int n = 2;
+  __CPROVER_assert(&((char *)base)[offset] != 0, "no wrap-around expected");
+}
```

Comment:
This one feels like it's a bit overfitted to the current encoding -- how about checking that the conversion to an integer doesn't get smaller?

Reply:
Done.